smart_campus/app/views.py

```diff
@@ -142,10 +279,10 @@ def get_all_stations(request):
             'content': station.content,
             'category': str(station.category),
             'location': station.location.get_coords(),
-            'image': [{'image_url': img.image.url, 'primary': img.is_primary}
+            'image': [{'image_url': 'http://{0}/{1}'.format(request.get_host(), img.image.url), 'primary': img.is_primary}
                       for img in station.stationimage_set.all()]
         }
         for station in Station.objects.all()
     ]
 
-    return JsonResponse({'status': 'true', 'message': 'Success', 'data': data})
+    return JsonResponse({'status': 'true', 'message': 'Success', 'data': data}, json_dumps_params={'ensure_ascii': False}, content_type='application/json; charset=utf-8')
```

Comment:
Make sure all other `JsonResponse`s that would contain Chinese are set as this one.